src/lib/feature-finder.js

```diff
@@ -19,15 +20,17 @@ export default function(cucumberOptions) {
   .then((files) => {
     files = _.flattenDeep(files);
     return files.map((file) => {
-      let featureData = parseFeature(file);
-      return featureData.children
+      let parsedData = parseFeature(file);
+      didDetectErrors = parsedData.didDetectErrors ? true : didDetectErrors;
```

Comment:
It's one variable for all feature files, which is why i am basically doing an OR with itself.